conans/client/tools/files.py

```diff
@@ -194,24 +194,30 @@ def emit(self, record):
     if not patchset:
         raise ConanException("Failed to parse patch: %s" % (patch_file if patch_file else "string"))
 
+    def decode_clean(path, prefix):
+        path = path.decode("utf-8").replace("\\", "/")
+        if path.startswith(prefix):
+            path = path[2:]
+        return path
+
+    def handle_null(path):
```

Comment:
Why the name `handle_null`? Shouldn't be something related with the `strip` parameter?